apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -368,6 +354,13 @@ export function AppRoot({
     electionDefinitionQuery.data,
   ]);
 
+  // reset the configuration mutation whenever USB is removed
+  useEffect(() => {
+    if (usbDrive.status !== 'mounted' && !configureMutation.isIdle) {
+      configureMutation.reset();
+    }
+  }, [configureMutation, usbDrive.status]);
+
```

Comment:
The idea of this commit is that we need to manage the configuration mutation more closely (to get our auto-configure and then hold off until you re-insert USB approach correct) _and_ use the mutation's state as a source of truth for component state.

Reply:
see: https://votingworks.slack.com/archives/C05BNBX5C3G/p1687800639587809
see: https://votingworks.slack.com/archives/C05BNBX5C3G/p1687884921336539

Reply:
@jonahkagan I was looking at the VxScan solution - does that work because the mutation resets when the unconfigured screen is unmounted?

Reply:
I'm not totally sure what you're asking. Maybe let's discuss?

My quick take (without having dug into this code), is that having a `useEffect` with `configureMutation` in the dependency array makes me nervous.

I think we want the same behavior for scan and central-scan, right? So I would guess that using the same logic from scan should work...

Reply:
The only difference between the two is that in VxCentralScan we show a sort of "success screen" afterwards whereas in VxScan I think we just jump to the next component. We could maintain the success screen with callbacks but I think it exists to remind the user to eject the USB, but moving forward VxCentralScan also needs continuous backup... so I'm in favor of just removing it.

Reply:
Sounds good to me, maybe worth talking through with @mattroe

Reply:
Sounds good. This thread is the biggest piece of feedback, I'll wait till I'm actually at my desk tomorrow to check in with folks and resolve. 

@kofi-q if you get to VxCentralScan UI work before this is merged, you may want to branch off of this PR.